src/shell-app-system.c

```diff
@@ -207,14 +207,6 @@ app_is_stale (ShellApp *app)
   return !is_unchanged;
 }
 
-static gboolean
-stale_app_remove_func (gpointer key,
-                       gpointer value,
-                       gpointer user_data)
-{
-  return app_is_stale (value);
```

Comment:
Squash this commit into https://github.com/endlessm/gnome-shell/pull/11/commits/d7372ddaf8e7ee753f2a919c3af7d43afa1d7ce3